docs/values-files/values-nginx.yaml

```diff
@@ -23,3 +23,6 @@ controller:
     requests:
       cpu: 100m
       memory: 300Mi
+
```

Comment:
This prevents the exposure of the same header on our customers since this file is part of the documentation we provide.